### PR TITLE
Issue-47 Introduce env.file

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: env.file [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# URL to use in GitHub PRs link
+JENKINS_URL=URL
+
+# Token used to publish checks in GitHub
+GITHUB_TOKEN=token
+
+# Token used to publish comment on branch build (not mandatory)
+#SLACK_TOKEN=token
+
+# Credentials used for publishing image on docker hub
```
Comment (mbaechler) [inline env.file]: I guess it's not mandatory neither

File: env.file [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# URL to use in GitHub PRs link
+JENKINS_URL=URL
+
+# Token used to publish checks in GitHub
```
Comment (mbaechler) [inline env.file:4]: not mandatory ?

File: env.file [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+GITHUB_TOKEN=token
+
+# Token used to publish comment on branch build (not mandatory)
```
Comment (rouazana) [inline env.file]: on master build?

File: env.file (hunk 1 above)
Comment (aduprat) [inline env.file]: Right

File: env.file (hunk 2 above)
Comment (aduprat) [inline env.file:4]: It's actually mandatory.

File: env.file (hunk 3 above)
Comment (aduprat) [inline env.file]: Right

**Comment by rouazana** [comment]
:+1:

**Comment by mbaechler** [comment]
:

**Comment by mbaechler** [comment]
:+1:

**Comment by chibenwa** [comment]
:+1:

**Pull request merged**